README.md

```diff
@@ -22,6 +22,11 @@ Here are some ideas to get you started:
 
 <p align="left"> <a href="https://x.com/shohamgilad" target="blank"><img src="https://img.shields.io/twitter/follow/shohamgilad?logo=x&style=for-the-badge" alt="shohamgilad" /></a> </p>
 
+- 👥 I'm leading 3 big tech communities in Israel:
```

Comment:
Remove the whatsapp links
Use the word transformation instead of revolution
About mcp community mention that's its a general ai community
Mentioned thousands of members in mcp/n8n community
And hundreds in the guild one